docs/Releases/v0.2.0-release.md

```diff
@@ -0,0 +1,364 @@
+>
+>```bash
+$  kubectl get bundlemetadata operatorhubio-argocd-operator.v0.6.0 -o yaml | grep image 
+  image: quay.io/operatorhubio/argocd-operator@sha256:1a9b3c8072f2d7f4d6528fa32905634d97b7b4c239ef9887e3fb821ff033fef6
+  - image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+  - image: quay.io/argoprojlabs/argocd-operator@sha256:99aeec24cc406d06d18822347d9ac3ed053a702d8419191e4e681075fed7b9bb
+  - image: quay.io/operatorhubio/argocd-operator@sha256:1a9b3c8072f2d7f4d6528fa32905634d97b7b4c239ef9887e3fb821ff033fef6
+```
+>
+>Looks like the last image reference is infact the one that is also visible in the `installedBundleResource` field. 
+>
+>We can also pin the installation of an operator to a specific version, if we wish to install something other than the latest: 
+>
+>```bash 
+$ kubectl apply -f - <<EOF
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: Operator
+metadata:
+  name: argocd-operator
+spec:
+  packageName: argocd-operator
+  version: v0.5.0
+EOF
+operator.operators.operatorframework.io/argocd-operator created
+>
+>$ kubectl get operator argocd-operator -o yaml | grep status -A 4
+status:
+  conditions:
+  - lastTransitionTime: "2023-06-21T15:32:54Z"
+    message: resolved to "quay.io/operatorhubio/argocd-operator@sha256:5f4533de114e9301e836155190dc2c66a969f55085af03a4959a2fb0af74e1f4"
+    observedGeneration: 1
+--
+    status: "True"
+    type: Resolved
+  - lastTransitionTime: "2023-06-21T15:33:03Z"
+    message: installed from "quay.io/operatorhubio/argocd-operator@sha256:5f4533de114e9301e836155190dc2c66a969f55085af03a4959a2fb0af74e1f4"
+    observedGeneration: 1
+--
+    status: "True"
+    type: Installed
+  installedBundleResource: quay.io/operatorhubio/argocd-operator@sha256:5f4533de114e9301e836155190dc2c66a969f55085af03a4959a2fb0af74e1f4
+  resolvedBundleResource: quay.io/operatorhubio/argocd-operator@sha256:5f4533de114e9301e836155190dc2c66a969f55085af03a4959a2fb0af74e1f4
+```
+>
+>We can confirm that the v0.5.0 version was in fact the version that was installed, by again matching the image reference visible in the `installedBundleResource` field with the image reference for v0.5.0 
+>
+>```bash
+k get bundlemetadata operatorhubio-argocd-operator.v0.5.0 -o yaml | grep image 
+  image: quay.io/operatorhubio/argocd-operator@sha256:5f4533de114e9301e836155190dc2c66a969f55085af03a4959a2fb0af74e1f4
+  - image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+  - image: quay.io/argoprojlabs/argocd-operator@sha256:2f5c0d4567607266ccacb91d8c2e3b18c2afe0edaf55855dcb1a06b02173b520
+  - image: quay.io/operatorhubio/argocd-operator@sha256:5f4533de114e9301e836155190dc2c66a969f55085af03a4959a2fb0af74e1f4
+```
+> In one of the upcoming releases, we plan on providing more straightforward way of gauging the information about which version of an operator has been installed. The discussion for that effort is being tracked in [this issue](https://github.com/operator-framework/operator-controller/issues/270). 
+
```

Comment:
Visual nit: IMO the "callout" functionality would look great for this note rather than just markdown quoting!